hub/scripts.js

```diff
@@ -245,6 +245,26 @@
     });
   }
 
+  /**
+   * Loads a JS file on the page
+   * @param {String} src The path to the JS file
+   * @param {Object} param The parameters used to load the file
+   */
+  function loadJS(src, param) {
+    if (isNonEmptyString(src)) {
+      const script = document.createElement('script');
+      script.setAttribute('src', src);
+      script.setAttribute('type', 'text/javascript');
+      if (typeof param === 'object') {
+        if (isNonEmptyString(param.id)) {
+          script.setAttribute('id', param.id);
```

Comment:
It looks like `param` could only set an ID. Would it be a good idea to iterate other its properties and set them on the `script` tag as well? Or, if you see it being used just to set an ID, maybe we can rename it to `id`?

Reply:
right now param is used only for ids, but we might need to add other properties in the future; I thought about keeping backwards compatibility.